components/util/opts.rs

```diff
@@ -167,6 +167,9 @@ pub struct Opts {
     /// Dumps the layer tree when it changes.
     pub dump_layer_tree: bool,
 
+    ///Record and write to disk each frame sent to webrender.
```

Comment:
Does this one do the similar thing as the one in `DebugOptions`? Then we should probably use the same comment.